pkg/imgpkg/cmd/push_test.go

```diff
@@ -56,7 +57,7 @@ func TestMultiImgpkgDirError(t *testing.T) {
 		t.Fatalf("Expected validations to err, but did not")
 	}
 
-	if !strings.Contains(err.Error(), "This directory contains multiple bundle definitions. Only a single instance of .imgpkg/images.yml can be provided") {
+	if !strings.Contains(err.Error(), fmt.Sprintf("This directory contains multiple bundle definitions. Only a single instance of .imgpkg%simages.yml can be provided", string(os.PathSeparator))) {
```

Comment:
im not sure how i feel about us flipping slashes in error messages. should we do this or just leave it as is?

Reply:
hmmm i'm not sure i fully understand here...

This is within a test asserting an error message though... so the implementation on windows is going to return a different error message when running this test in linux (the diff being the slash). So this aims to make the test make the right assertion based on the OS that is running said test.

Reply:
i see, we are generating that stuff on the fly.